tests/test_storageclass_invalid.py

```diff
@@ -1,19 +1,95 @@
 import logging
 import pytest
+import os.path
+import yaml
 
 from resources.pvc import PVC
 from tests import helpers
 from ocs import constants, defaults
 from ocs.exceptions import TimeoutExpiredError
 from ocsci.testlib import tier3, ManageTest
+from resources.ocs import OCS
 
 
 logger = logging.getLogger(__name__)
 
 
+@pytest.fixture(
+    params=[pytest.param({
+            'template_dir': constants.TEMPLATE_CSI_FS_DIR,
+            'values': {
+                'storageclass_name': 'invalid-storageclass',
+                'provisioner': "invalid_provisioner",
+                'monitors': 'invalid_monitors',
+                'provision_volume': "invalid_provisioner_volume",
+                'ceph_pool': 'invalid_pool',
+                'root_path': 'invalid_root_path',
+                'provisioner_secret_name': 'invalid_provisioner_secret_name',
+                'provisioner_secret_namespace': 'invalid_provisioner_secret_namespace',
+                'node_stage_secret_name': 'invalid_node_stage_secret_name',
+                'node_stage_secret_namespace': 'invalid_node_stage_secret_namespace',
+                'mounter': 'invalid_mounter',
+                'reclaim_policy': 'Delete'
+            }
+            },
+        marks=pytest.mark.ocs_331),
```

Comment:
```suggestion
        marks=pytest.mark.polarion_id("OCS-331")),
```

I have checked how to align the markers here with https://github.com/red-hat-storage/ocs-ci/pull/243/ and it turns out that suggestion above will work just fine:

```
$ python3 -m pytest .  -v --junit-xml=report.xml && xmllint --format report.xml
=========================================================== test session starts ===========================================================
platform linux -- Python 3.7.3, pytest-3.9.3, py-1.7.0, pluggy-0.8.1 -- /usr/bin/python3
cachedir: .pytest_cache
rootdir: /home/martin/tmp/parametrize-polarion-id, inifile: pytest.ini
plugins: pyfakefs-3.1
collected 2 items                                                                                                                         

test_foo.py::test_storageclass_invalid[CephFS] PASSED                                                                               [ 50%]
test_foo.py::test_storageclass_invalid[RBD] PASSED                                                                                  [100%]

--------------------------------- generated xml file: /home/martin/tmp/parametrize-polarion-id/report.xml ---------------------------------
======================================================== 2 passed in 0.02 seconds =========================================================
<?xml version="1.0" encoding="utf-8"?>
<testsuite errors="0" failures="0" name="pytest" skips="0" tests="2" time="0.022">
  <testcase classname="test_foo" file="test_foo.py" line="64" name="test_storageclass_invalid[CephFS]" time="0.00044226646423339844">
    <properties>
      <property name="polarion-testcase-id" value="OCS-331"/>
    </properties>
  </testcase>
  <testcase classname="test_foo" file="test_foo.py" line="64" name="test_storageclass_invalid[RBD]" time="0.0003726482391357422">
    <properties>
      <property name="polarion-testcase-id" value="OCS_341"/>
    </properties>
  </testcase>
</testsuite>
```

(the output above was tested on a toy example, so that I don't have to redeploy actual cluster)

Reply:
Done